### PR TITLE
Abwl sthinbetween 20211705

### PR DESCRIPTION
**Pull request by sthinbetween**

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by ZeroPointMax** [comment]
Haha, you have been fooled!

In deiner Abwesenheit haben wir einen Linting-Check installiert, der sich jetzt über dein Markdown beschwert :D

**Comment by sthinbetween** [comment]
> 
> 
> Haha, you have been fooled!
> 
> In deiner Abwesenheit haben wir einen Linting-Check installiert, der sich jetzt über dein Markdown beschwert :D

Da bin ich einmal ein halbes Semester nicht da.

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: ABWL-LEDER/README.md [hunk 1]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
 
 # Externes Rechnungswesen
 
+- Personalführung selbst ist die individuelle Führung des Menschen und daher nicht digitalisierbar
+- externes Rechnungswesen ist alles, was das Unternehmen nach außen hin verlässt
+  - z.B. Nachweise über den Erfolg des Unternehmens
+
+## umfasst:
```
Comment (ZeroPointMax) [inline ABWL-LEDER/README.md:422]: Super, noch mehr Linting Fehler...

@sthinbetween kann ich in demselben Ruck auch sowas hier umwandeln:

```suggestion
**umfasst:**
```

**Review by nukerxy — DISMISSED** [review]
Inhalt und Rechtschreibung: approve

File: ABWL-LEDER/README.md [hunk 2]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+  - z.B. Dividende bei Aktiengesellschaften
+- im Rahmen der Insolvenz
+  - am Markt immer Risiko vorhanden, welches abgeschätzt werden muss
+  - Verpflichtungen ggü. Lieferanden und Forderungen und Verbindlichkeiten ggü. Kunden
+  - wenn dieses Verhältnis in Schieflage gerät ist die Bilanz ist ein Signal für Überschuldung
+- Entscheidungsanreizsystem
+  - gute Bilanz kann Entscheidungsgrundlage für Investoren sein
+  - Gewinnentwicklung als Signal für Investoren
+
+## Kriterien:
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:451]:
```suggestion
## Kriterien d. externen Rechnungswesen
```

File: ABWL-LEDER/README.md [hunk 3]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+  - mit welchen Kosten sind wir in der Lage welche Leistung zu erzeugen?
+  - welchen Preis und Menge können wir am Markt erzielen?
+- Selbstkosten der Produkte und Dienstleistungen
+  - Rahmenkosten werden außen vor gelassen
+  - daraus folgt: Preiskalkulation der Produkte und Dienstleistungen
+  - Preis der Produkte wird vom Markt bestimmt
+    - wenn Marktpreis höher als Kosten für Produkte: Gewinn
+    - andernfalls müssen Kosten gesenkt werden um keinen Verlust zu machen
+
+## Kriterien:
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:559]:
```suggestion
## Kriterien d. internen Rechnungswesens
```

File: ABWL-LEDER/README.md [hunk 4]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+  - Darstellen des Erfolgs des Unternehmens
+  - Informationsquelle ist die GuV
+
+**Einzelabschluss** ist der Jahresabschluss eines rechtlich selbstständigen Unternehmens
+
 # Internes Rechnungswesen
 
+Beschreibt Steuerung und Regelung der Prozesse im Unternehmen und hängt über Schnittstellen zusammen mit dem externen Rechnungswesen 
+
+## umfasst:
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:547]:
```suggestion
```
einfach weg damit?

File: ABWL-LEDER/README.md (hunk 1 above)
Comment (nukerxy) [inline ABWL-LEDER/README.md:422]:
```suggestion
```

File: ABWL-LEDER/README.md [hunk 5]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+  - ist vergangenheitsbasiert und dient möglicher Prognose
+  - meist öffentlich einsehbar (z.B. Bundesanzeiger)
+- Ausschüttungsbemessungsfunktion
+  - Gewinnermittlung
+  - Gewinnverwendung
+  - z.B. Dividende bei Aktiengesellschaften
+- im Rahmen der Insolvenz
+  - am Markt immer Risiko vorhanden, welches abgeschätzt werden muss
+  - Verpflichtungen ggü. Lieferanden und Forderungen und Verbindlichkeiten ggü. Kunden
+  - wenn dieses Verhältnis in Schieflage gerät ist die Bilanz ist ein Signal für Überschuldung
```
Comment (RvNovae) [inline ABWL-LEDER/README.md:444]:
```suggestion
  - wenn dieses Verhältnis in Schieflage gerät ist die Bilanz ein Signal für Überschuldung
```

File: ABWL-LEDER/README.md [hunk 6]
```diff
@@ -48,7 +48,17 @@ Allgemeine Betriebswirtschaftslehre
...
 - [Externes Rechnungswesen](#externes-rechnungswesen)
+  - [umfasst:](#umfasst)
+  - [Funktionen:](#funktionen)
+  - [Kriterien:](#kriterien)
+  - [Wer braucht was?](#wer-braucht-was)
+  - [Was ist was?](#was-ist-was)
+  - [Inhaltliche Bedeutung der Teile des Jahresabschlusses](#inhaltliche-bedeutung-der-teile-des-jahresabschlusses)
 - [Internes Rechnungswesen](#internes-rechnungswesen)
+  - [umfasst:](#umfasst-1)
+  - [Kriterien:](#kriterien-1)
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:59]:
```suggestion
  - [Kriterien d. externen Rechnungswesens:](#kriterien-1)
```

File: ABWL-LEDER/README.md [hunk 7]
```diff
@@ -48,7 +48,17 @@ Allgemeine Betriebswirtschaftslehre
     - [Methods-Time-Measurement-Analysis](#methods-time-measurement-analysis)
     - [REFA](#refa)
 - [Externes Rechnungswesen](#externes-rechnungswesen)
+  - [umfasst:](#umfasst)
+  - [Funktionen:](#funktionen)
+  - [Kriterien:](#kriterien)
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:53]:
```suggestion
  - [Kriterien d. internen Rechnungswesens: ](#kriterien)
```

File: ABWL-LEDER/README.md [hunk 8]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+- Beispiel:
+  - Abverkauf von Flächen
+
+Eigenkapitalspiegel
+
+- stellt neben dem Unternehmenserfolg auch alle anderen Eigenkapitalveränderungen während einer Rechnungsperiode dar
+
+Segmentberichterstattung
+
+- Ergänzung der Rechnungslegung um dissaggierte finanzielle und nicht finazielle Daten zu Geschäftsfeldern, Regionen und anderen Kriterien
```
Comment (RvNovae) [inline ABWL-LEDER/README.md:517]: Was bedeutet `diassggiert`?

File: ABWL-LEDER/README.md [hunk 9]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+- Ergänzung der Rechnungslegung um dissaggierte finanzielle und nicht finazielle Daten zu Geschäftsfeldern, Regionen und anderen Kriterien 
+- Einblick in die Chancen und Risiken der einzelnen Geschäftsfelder wird verbessert
+
+Lagebericht
+
+- verbale Darstellung der Unternehmensführung über bestimmte Bereiche
+
+## Inhaltliche Bedeutung der Teile des Jahresabschlusses
+
+- Finanzlage $\rightarrow$ sämtliche Aspekte der Finazierung
```
Comment (RvNovae) [inline ABWL-LEDER/README.md:526]:
```suggestion
- Finanzlage $\rightarrow$ sämtliche Aspekte der Finanzierung
```

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: ABWL-LEDER/README.md [hunk 10]
```diff
@@ -405,8 +415,175 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+
+- **Ertragslage**
+  - Darstellen des Erfolgs des Unternehmens
+  - Informationsquelle ist die GuV
+
+**Einzelabschluss** ist der Jahresabschluss eines rechtlich selbstständigen Unternehmens
+
 # Internes Rechnungswesen
 
+Beschreibt Steuerung und Regelung der Prozesse im Unternehmen und hängt über Schnittstellen zusammen mit dem externen Rechnungswesen
```
Comment (RvNovae) [inline ABWL-LEDER/README.md:543]:
```suggestion
beschreibt Steuerung und Regelung der Prozesse im Unternehmen und hängt über Schnittstellen zusammen mit dem externen Rechnungswesen 
```

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Review by RvNovae — APPROVED** [review]
Vielen Dank.

File: ABWL-LEDER/README.md (hunk 10 above)
Comment (nukerxy) [reply ABWL-LEDER/README.md]: > Vielen Dank.

Boah das habe ich alles übersehen, haha

File: ABWL-LEDER/README.md (hunk 8 above)
Comment (ZeroPointMax) [reply ABWL-LEDER/README.md:517]: Gute Frage

Comment (sthinbetween) [reply ABWL-LEDER/README.md:517]: Hab ich mich beim Mitschreiben schon gefragt

File: ABWL-LEDER/README.md [hunk 11]
```diff
@@ -405,8 +413,174 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+  - z.B. Dividende bei Aktiengesellschaften
+- im Rahmen der Insolvenz
+  - am Markt immer Risiko vorhanden, welches abgeschätzt werden muss
+  - Verpflichtungen ggü. Lieferanden und Forderungen und Verbindlichkeiten ggü. Kunden
+  - wenn dieses Verhältnis in Schieflage gerät ist die Bilanz ist ein Signal für Überschuldung
+- Entscheidungsanreizsystem
+  - gute Bilanz kann Entscheidungsgrundlage für Investoren sein
+  - Gewinnentwicklung als Signal für Investoren
+
+## Kriterien d. externen Rechnungswesens:
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:449]:
```suggestion
## Kriterien d. externen Rechnungswesens: 
```

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by ZeroPointMax** [comment]
ffs hat der Linter Crack geraucht?

**Comment by nukerxy** [comment]
Jetzt sind wir hier doch wirklich mit den Reviews kollidiert...

File: ABWL-LEDER/README.md [hunk 12]
```diff
@@ -405,8 +413,174 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+    - wenn Marktpreis höher als Kosten für Produkte: Gewinn
+    - andernfalls müssen Kosten gesenkt werden um keinen Verlust zu machen
+
+## Kriterien d. internen Rechnungswesens
+
+- **Entscheidungsgrundlage für** Unternehmensinterne, Management, Betriebsrat, Mitarbeiter
+- **Ziel:** Planung und Steuerung der betrieblichen Prozesse (meist mittels ERP-System)
+- **gesetzliche Regelung:** Grundsatz ist die doppelte Buchführung, Einzelvorschriften in der Darstelllung (z.B. Abschreibungen)
+- **Objekt der erfassten Daten:** von einzelnen Unternehmensbereichen bis zum Gesamtunternehmen
+- **Zeitraum der erfassten Daten:**
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:562]:
```suggestion
- **Zeitraum der erfassten Daten:**
```

**Comment by ZeroPointMax** [comment]
> Jetzt sind wir hier doch wirklich mit den Reviews kollidiert...

Oha, sorry

File: ABWL-LEDER/README.md (hunk 12 above)
Comment (nukerxy) [reply ABWL-LEDER/README.md:562]: This rule is triggered on any lines that end with whitespace. To fix this, find the line that is triggered and remove any trailing spaces from the end.

File: ABWL-LEDER/README.md [hunk 13]
```diff
@@ -405,8 +413,174 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+- **Entscheidungsgrundlage für** Unternehmensinterne, Management, Betriebsrat, Mitarbeiter
+- **Ziel:** Planung und Steuerung der betrieblichen Prozesse (meist mittels ERP-System)
+- **gesetzliche Regelung:** Grundsatz ist die doppelte Buchführung, Einzelvorschriften in der Darstelllung (z.B. Abschreibungen)
+- **Objekt der erfassten Daten:** von einzelnen Unternehmensbereichen bis zum Gesamtunternehmen
+- **Zeitraum der erfassten Daten:** 
+  - unterjährig $\rightarrow$ Kostenleistungsrechnung (KLR)
+  - mehrjährig $\rightarrow$ Investitionsrechnung (Abschreibung gesamt)
+- **Instrumente:** KLAR, Investitionsrechnung, Finanzrechnung (Bonität)
+
+## Balance Score Card
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:567]:
```suggestion
## Balance Score Card
```

File: ABWL-LEDER/README.md [hunk 14]
```diff
@@ -405,8 +413,174 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+
+<!--3 Vor- und Nachteile müssen gewusste werden-->
+<!--TODO: von Folien übernehmen-->
+
+## Kosten- Leistungsrechnung
+
+Vollkostenrechnung
+
+- ca. 1950 entwickelt
+- Erfassung aller Kosten und deren Zuordnung zum Produkt oder Dienstleistung
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:577]:
```suggestion
- Erfassung aller Kosten und deren Zuordnung zum Produkt oder Dienstleistung 
```

**Review by ZeroPointMax — CHANGES_REQUESTED** [review]
Ich blockiere mal aufgrund der offenen Baustellen

File: ABWL-LEDER/README.md [hunk 15]
```diff
@@ -405,8 +413,174 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+  - Bilanz
+  - GuV
+  - Anhang
+  - Kapitalflussrechnung
+  - Eigenkapitalspiegel
+  - *freiwillig* eine Segmentberichterstattung
+  - ergänzt um Lagebericht
+  - definiert in §244 HGB
+
+## Was ist was?
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:484]:
```suggestion
## Was ist was
```

Comment (nukerxy) [reply ABWL-LEDER/README.md:484]: This rule is triggered on any header that has a punctuation character as the last character in the line

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- ABWL-LEDER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: ABWL-LEDER/README.md [hunk 16]
```diff
@@ -405,8 +413,174 @@ Sollzeitanalyse = Vergleich der SOLL- und gemessenen IST-Zeiten
...
+
+<!--3 Vor- und Nachteile müssen gewusste werden-->
+<!--TODO: von Folien übernehmen-->
+
+## Kosten- Leistungsrechnung
+
+Vollkostenrechnung
+
+- ca. 1950 entwickelt
+- Erfassung aller Kosten und deren Zuordnung zum Produkt oder Dienstleistung
```
Comment (nukerxy) [inline ABWL-LEDER/README.md:577]:
```suggestion
- Erfassung aller Kosten und deren Zuordnung zum Produkt oder Dienstleistung
```

**Comment by ZeroPointMax** [comment]
Ich glaube wir haben's, Freunde der Sonne und des Mondes

Ich würde ein Squash'n'Merge vorschlagen

**Review by ZeroPointMax — APPROVED** [review]
Vielen Dank, Ben ※\\(^o^)/※

**Comment by ZeroPointMax** [comment]
Und alle, die mitgewirkt haben :)

**Review by nukerxy — APPROVED** [review]

**Pull request merged**